.gitignore

```diff
@@ -1,15 +1,17 @@
 [bB]uild/*
+[bB]uild.xcode
 CMakeLists.txt.user*
 *~
 *.mo
 dylibbundler.log
 performous-app-build.sh
```

Comment:
is this correct? This is set to ignore but the script is checked in.

Reply:
It's not the script. I initially made a vlrtualenv inside that folder.

Reply:
Not but I think it won't hurt to keep it for a bit.